src/volttron/types/auth/authz_types.py

```diff
@@ -8,30 +8,59 @@
 RPC_CAPABILITIES = "rpc_capabilities"
```

Comment:
Why aren't we using an enumerated value rather than just constants?  Does the converter not allow enumerations?